.github/ISSUE_TEMPLATE/update-images.md

```diff
@@ -19,18 +19,23 @@ Reference to the previous issue for updating images: <!-- (issue reference) -->
 
 Action items:
 
-- [ ] Update the issue template to add/remove [images](https://github.com/kubermatic/kubeone/blob/main/pkg/templates/images/images.go) as appropriate
+- [ ] Update the issue template to add/remove [images](https://github.com/kubermatic/kubeone/blob/main/pkg/templates/images/images.go) as appropriate <!-- (PR reference|already the latest) -->
 
 The following components/images should be updated:
 
 ### General
 
+- [ ] [machine-controller](https://github.com/kubermatic/machine-controller) <!-- (PR reference|already the latest) -->
+- [ ] [operating-system-manager](https://github.com/kubermatic/operating-system-manager)
+- [ ] [metrics-server](https://github.com/kubernetes-sigs/metrics-server) <!-- (PR reference|already the latest) -->
+- [ ] [NodeLocalDNS](https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dns/nodelocaldns/nodelocaldns.yaml) <!-- (PR reference|already the latest) -->
+- [ ] [Cluster Autoscaler](https://github.com/kubernetes/autoscaler) <!-- (PR reference|already the latest) -->
+
+### CNIs
+
 - [ ] [Canal CNI](https://github.com/projectcalico/calico) <!-- (PR reference|already the latest) -->
 - [ ] [Calico VXLAN CNI](https://github.com/projectcalico/calico) <!-- (PR reference|already the latest) -->
```

Comment:
Should Calico VXLAN CNI be removed from the list? Iirc we discussed in Slack that this is not required for updates anymore

Reply:
I left it as of now until we don't remove the addon itself.